play/src/front/Components/ActionBar/AvailabilityStatus/Interfaces/AvailabilityStatusPropsInterface.ts

```diff
@@ -0,0 +1,20 @@
+import { AvailabilityStatus } from "../../../../../../../libs/messages";
```

Comment:
use import "@workadventure/messages" because when we will compile that, it won't work